Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -1,374 +1,207 @@
 /* Style applied to the DropDownMenu. */
 
-span.DropDownMenuContainer 
+span.DropDownMenuContainer
 {
-  display: block;
+  display: flex;
+  align-items: center;
   position: relative;
-  font-family: sans-serif;
+  height: var(--remotion-themed-min-height);
   width: 100%;
-  min-width: 18px;
-}
-
-tr.bocListDataRow span.DropDownMenuContainer,
-tr.bocListDataRowSelected span.DropDownMenuContainer
-{
-  display: inline-block;
 }
 
 span.DropDownMenuSelect
 {
-  display: block;
+  display: inline-flex;
   position: relative;
-  border: solid 1px #AAAAAA;
-  text-align: left;
-  vertical-align: top;
+  line-height: var(--remotion-themed-line-height);
+  width: 100%;
+  border: var(--remotion-themed-border);
+  border-radius: var(--remotion-themed-border-radius);
   white-space: nowrap;
-  cursor: hand;
   cursor: pointer;
-  line-height: 1.25em;
-  height: 1.4em;
 }
 
-td.buttonCell span.DropDownMenuSelect
+div.DropDownMenuOptions
 {
-  /* only when display just arrow */
-  display: inline-block;
+  /* int max for highest z-index possible */
+  z-index: 2147483647;
+  position: absolute;
 }
 
-span.DropDownMenuSelect.disabled
+ul.DropDownMenuOptions
```

Comment:
since you moved the options down, please move the rest of the option styles as well.

Reply:
Currently, it is ordered by specificity, so moving it down would disturb that order. If it should still be moved due to comprehensibility, I will reorganize it.